README.adoc

```diff
@@ -7,7 +7,7 @@ This repository hosts the documentation sources including translations (if
 available).
 
 Released versions of the documentation are published at
-https://www.suse.com/documentation/.
+https://documentation.suse.com/sles-sap/.
```

Comment:
Just a quick question: As we've got also aliases (doc.suse.com, docs.suse.com), which is the preferred link nowadays? The abbreviated version of `doc[s]?` is shorter and might potentially give better linebreaks in PDF, right?

I think, docs.suse.com looks also a bit smarter and the "docs" prefix is frequently used today as you can see it in docs.python.com, docs.travis-ci.com, and probably a few others.

Additionally, I would recommend to add the preferred URL of the documentation link into our styleguide, but that's another issue. :wink:

Reply:
The aliases don't exist yet. And they will really only be aliases, redirecting to documentation.suse.com, so I think documentation.suse.com will remain the address we use everywhere.

Reply:
I see, thanks. Then lets document this in our styleguide. I've opened SUSE/doc-styleguide#94 for this purpose.

Reply:
Thank!

Reply:
I will take care of integrating it into the other branches.